### PR TITLE
Tasks, AddTask, EditTask page objects implementation

### PR DESCRIPTION
**Pull request by DenisPitsul**

### CODE REVIEW

**Review by Maksym637 — APPROVED** [review]
good

File: src/main/java/org/ssu/edu/teachua/ui/components/date_picker/SelectDateComponent.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    @FindBy(how = How.XPATH, using = ".//button[@class='ant-picker-header-super-next-btn']")
+    private WebElement nextYearBtn;
+
+    public SelectDateComponent(WebDriver driver, WebElement node) {
+        super(driver, node);
+    }
+
+    public void selectDate(int day, int month, int year) {
+        if (day > 31 || day < 1 || month > 12 || month < 1 || year < 1)
+            return;
```
Comment (lhalam) [inline src/main/java/org/ssu/edu/teachua/ui/components/date_picker/SelectDateComponent.java:42]: remove this lines

File: src/main/java/org/ssu/edu/teachua/ui/components/date_picker/SelectDateComponent.java [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        months.put("May", 5);
+        months.put("Jun", 6);
+        months.put("Jul", 7);
+        months.put("Aug", 8);
+        months.put("Sep", 9);
+        months.put("Oct", 10);
+        months.put("Nov", 11);
+        months.put("Dec", 12);
+        return months;
+    }
```
Comment (lhalam) [inline src/main/java/org/ssu/edu/teachua/ui/components/date_picker/SelectDateComponent.java:98]: move to attr

File: src/main/java/org/ssu/edu/teachua/ui/pages/tasks/EditTaskPage.java [hunk 3]
```diff
@@ -1,10 +1,55 @@
...
+
+    @Override
+    public AddTaskPage typeDescription(String title) {
+        descriptionInput.clear();
+        return super.typeDescription(title);
+    }
+
+    @Override
+    public AddTaskPage selectChallenge(String challenge) {
+        Actions act =  new Actions(driver);
```
Comment (lhalam) [inline src/main/java/org/ssu/edu/teachua/ui/pages/tasks/EditTaskPage.java:50]: move to BAse

**Review by lhalam — APPROVED** [review]

**Pull request merged**